tests/io/vasp/test_outputs.py

```diff
@@ -831,9 +833,20 @@ def test_parse_potcar_cwd_relative(self):
                 assert vrun.potcar_spec[ipot]["summary_stats"] == potcar[ipot]._summary_stats
 
 
-class TestOutcar(PymatgenTest):
```

Comment:
I have to avoid inheriting from `PymatgenTest` as [`TestCase` (and its subclasses) currently don't support parametrize](https://github.com/pytest-dev/pytest/issues/10474), I believe this wouldn't affect this test as no `PymatgenTest/TestCase` specific method is used here.